frontend/templates/views/partials/first_survey.j2

```diff
@@ -0,0 +1,32 @@
+{% set survey_questions = [{
```

Comment:
No need to localize them just yet?

Reply:
haven't sent the questions to the localization folks

Reply:
will add it seperatly